packages/cli/src/commands/setup/auth/providers/dbAuth.webAuthn.js

```diff
@@ -20,7 +20,7 @@ export const config = {
 
 // required packages to install
 export const webPackages = ['@simplewebauthn/browser']
-export const apiPackages = ['@simplewebauthn/server']
+export const apiPackages = ['@simplewebauthn/server', 'base64url']
```

Comment:
I think we also need to add base64url to the normal dbauth one, because `dbAuthHandler` imports it (which is used in both regular dbAuth and webAuthN)

Reply:
Ahh good catch